src/hooks/useBreakpoint.tsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { BreakpointKey, Breakpoints } from "../Css";
+
+/**
+ * A React hook to return a record of responsive breakpoints that updates on resize.
+ *
+ * @example
+ * const { breakpoints } = useBreakpoint();
+ * if(breakpoints.mdAndDown) {...do something cool}
+ */
+
+type BreakpointsType = Record<BreakpointKey, boolean>;
+
+export function useBreakpoint(): { breakpoints: BreakpointsType } {
+  const [breakpoints, setBreakpoints] = useState({} as BreakpointsType);
```

Comment:
Thought: It probably doesn't matter, but the `{} as BreakpointsType` means that, until the `useEffect` runs, the very first caller will see `bp.mdAndDown` as `undefined` instead of `false`.

Makes me think to either:

a) Update the type to be `type BreakpointsType = Partial<Record<...>>` so that we're just explicit to the caller that `{}` is a value we'll be using (which seems fine, I don't think it would break the caller's `if (bp.print)` type checks, or

b) Initialize this `useState` with `{ mdAndDown: false, ... }` explicitly.

I'm generally ambivalent between the two, but maybe pick one? And, agreed, it's not a bug per se, but just something I could see be surprising for users if, for whatever reason, they venture outside the stock `if (bp.print)` usage and think like "oh I'm going to Object.values(bp)" or something.